src/lavinmq/auth/user.cr

```diff
@@ -14,131 +11,17 @@ module LavinMQ
       forward_missing_to @cache
     end
 
-    class User
-      include SortableJSON
-      getter name, password, permissions
-      property tags, plain_text_password
+    abstract class User
```

Comment:
Call this BasicUser or BaseUser, and we will have way less files/lines that has to be changed

Reply:
Maybe another name for this user then? LocalUser, PasswordUser? 
Edit: ah i see carl had the same comment further down